src/base/NekInterface.C

```diff
@@ -633,9 +633,10 @@ copyDeformationToDevice()
 }
 
 double
-sideMaxValue(const std::vector<int> & boundary_id, const field::NekFieldEnum & field)
+sideMaxValue(const std::vector<int> & boundary_id, const field::NekFieldEnum & field,
+             const MultiMooseEnum & pp_mesh)
 {
-  mesh_t * mesh = entireMesh();
+  mesh_t * mesh = (pp_mesh.size()>1) ? entireMesh() : flowMesh();
```

Comment:
Maybe we can do the MooseEnum options as `fluid, solid, all`  if that's easier to deal with. I'll leave that up to you

Reply:
My bad, I thought a MultiMooseEnum wouldn't allow `fluid fluid` as an input. And I like having inputs as `fluid solid all` a lot, I'll change to that.

Reply:
Honestly I thought it also would not allow it as well, but I thought I'd test it and `fluid fluid` gave 2x the fluid volume. I like the `fluid solid all` option better as well, now that I think about it. You'll want to use just a regular `MooseEnum` so that we're sure the user can only put in one of them at a time. Otherwise hopefully it's straightforward to swap.